Chat App/Chat App/Switcher/SwitcherView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SwitcherView.swift
+//  Chat App
+//
+//  Created by tornike on 29.04.23.
+//
+
+import UIKit
+
+enum Button {
+    case light
```

Comment:
we have Button in swiftui, and bad enum naming in general. Pls name it more self-explanatory